### ISSUE TITLE
[Generic] When UML says real, XSD has multiple sub types

### ISSUE THREAD

**Issue by JulieMoncorger** [betagouv/SPPNautInterface#233]
When UML says real, in XSD : 
- [noticeTime.noticeTimeHours](https://github.com/betagouv/SPPNautS1xyModelisation/blob/6dc55b14127c19cd0f722eb292668fa033811224/S127/XSD/S127/1.0.0/20190628/S127.xsd#L2639) is `xs:decimal`
- [underkeelAllowance.underkeelAllowanceFixed](https://github.com/betagouv/SPPNautS1xyModelisation/blob/6dc55b14127c19cd0f722eb292668fa033811224/S127/XSD/S127/1.0.0/20190628/S127.xsd#L2737) is `NonNegativeDecimal.1`
- [RadioCallingInPoint.orientationValue](https://github.com/betagouv/SPPNautS1xyModelisation/blob/6dc55b14127c19cd0f722eb292668fa033811224/S127/XSD/S127/1.0.0/20190628/S127.xsd#L634) is `Decimal0.0To360.0`

**Comment by stlejer-morvrini** [comment]
agreed. However by default we shall stick to the type specified in XSD.
I'd suggest we keep a list of these imprecise UML specs for me to push the issue to IHO.

**Comment by rik** [comment]
Yes, we've implemented the most specific type, as expressed by the XSD.

I don't think we should maintain a list because we've seen the issue in every UML diagram for S127. And we've also seen it in S131. It seems all UML diagrams are under-representing number constraints.

**Comment by rik** [comment]
This is the same issue for Integer: The UML only express Integer or Real while the XSD expresses more constraints (like Positive or a range).
